src/main/kotlin/persistence/data/Api.kt

```diff
@@ -0,0 +1,24 @@
+package persistence.data
+
+import data.Language
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+class Api {
+    private val door43Api: Door43Api
+
+    init {
+        val retrofit = Retrofit.Builder()
+                .baseUrl("https://td.unfoldingword.org/")
```

Comment:
lets keep strings like this in a config file (resources file)

Reply:
changed